content/11.interpretation.md

```diff
@@ -1,18 +1,18 @@
-However, in doing this, we need to be careful to correctly interpret the model's predictions.
+However, in doing this, we need to be careful to interpret the model's predictions correctly.
 Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to overinterpret the predictions in indulgent and/or inaccurate ways.
-As the classic statistical saying "correlation doesn't mean causation" implies, predictions by deep learning models don't necessarily speak to certain causual relationships.
-While we generally know this, and understand that accurately predicting an outcome doesn't imply the learning of any causal mechanism, it can be easy to forget this lesson when the predictions are extremely accurate.
+In accordance with the classic statistical saying, "correlation doesn't imply causation," predictions by deep learning models don't necessarily speak to certain causal relationships.
+While we generally know this and understand that accurately predicting an outcome doesn't imply the learning of any causal mechanism, it can be easy to forget this lesson when the predictions are extremely accurate.
 A poignant example of this lesson is from work where authors evaluated the capacities of several models to predict the probability of death for patients with pneumonia admitted to an intensive care unit [@tag:predicting-pneumonia-mortality; @doi:10.1145/2783258.2788613].
 Unsurprisingly, the neural network model achieved the best predictive accuracy.
-However, after fitting a rule-based model in order to better understand the relationships inherent to their data, the authors discovered that the hospital data implied the rule `HasAsthma(x) => LowerRisk(x)`.
+However, after fitting a rule-based model to understand the relationships inherent to their data better, the authors discovered that the hospital data implied the rule "$\text{HasAsthma}(x) \Rightarrow \text{LowerRisk}(x)$."
 This rule contradicts medical understanding, as having asthma doesn't make pneumonia better!
-Nonetheless, this rule was supported by the data, as pneumonia patients with a history of asthma tended to receive more aggressive care.
-The neural network had therefore also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
-Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.
+Nonetheless, the data supported this rule, as pneumonia patients with a history of asthma tended to receive more aggressive care.
+The neural network had, therefore, also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
+According to the predictions of the neural network, guiding treatment decisions would have been disastrous, even though the neural network had high predictive accuracy.
```

Comment:
The change to this sentence doesn't read right.

I would suggest leaving the origin, or the following if a less "passive voice" option is desired.

```suggestion
Therefore, it would have been disastrous to guide treatment decisions according to the predictions of the neural network, even though the neural network had high predictive accuracy.
```

Reply:
This is much better. Thanks!